soccer/gameplay/tactics/defense.py

```diff
@@ -403,6 +413,10 @@ def recalculate_threat_shot(threat_index):
         # print("Unused handlers: " + str(unused_threat_handlers))
         # print("---------------------")
 
+        # If we have nothing to block, bail
+        if not threats:
+            return
+
         smart = False
         if not smart:
```

Comment:
This was there previously, and I'm not entirely sure what it does. I think that when we restructure the defense play (if we do) this will be fixed :)